index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Bowling</title>
+    <link rel="stylesheet" href="public/normalize.css" />
+    <link rel="stylesheet" href="public/style.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Bowling Scores</h1>
+      <div id="final" class="final"></div>
+    </header>
+    <div class="scores">
+      <div class="frame">
+        <div class="number">1</div>
+        <input type="text" class="first" />
+        <input type="text" value="" maxlength="2" class="second" />
+        <div class="total"></div>
+      </div>
```

Comment:
I love the consistency of the HTML: the repeated structure and the minimum number of names.